sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -114,7 +115,8 @@ EnvironmentLogLevelListener::GetLogListener()
                   << Azure::DateTime(std::chrono::system_clock::now())
                          .ToString(
                              DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits)
-                  << "] " << LogLevelToConsoleString(level) << " : " << message << std::endl;
+                  << " T: " << std::this_thread::get_id() << "] " << LogLevelToConsoleString(level)
```

Comment:
Lets seperate this out into its own change.  If we want structured logging we should document the pattern and go 100% with it.  To that note, Jeff was against structured logging.

Reply:
I'm ok with reverting this, but I'll point out that this is just adding the thread ID to our existing "structured logging".